ironmon_tracker/Main.lua

```diff
@@ -0,0 +1,667 @@
+Main = {}
+
+-- The latest version of the tracker. Should be updated with each PR.
+Main.Version = { major = "7", minor = "0", patch = "0" }
+-- TODO: Purposefully downgrading Tracker version to test auto-update code, the real version should get updated to 7.0.0 before official release
```

Comment:
small note to remove this todo comment

Reply:
Lol whoops. Good note. I'll save this for after 7.0.0 release since this has no functional impact. Thanks.